docs/docs/creating-a-sitemap.md

```diff
@@ -2,7 +2,26 @@
 title: Creating a sitemap
 ---
 
-This is a stub. Help our community expand it.
+A sitemap can help web crawlers and search engines to quickly determine the pages that exist on your site. `gatsby-plugin-sitemap` is a plugin that automatically generates a sitemap for you during `gatsby build`.
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+## Install gatsby-plugin-sitemap
+
+`npm install --save gatsby-plugin-sitemap`
+
+## How to Use
+
+```javascript{3,6}:title=gatsby-config.js
+module.exports = {
+  siteMetadata: {
+	siteUrl: `https://www.example.com`,
```

Comment:
formatting is a bit off here - it should be indented more

Reply:
I can definitely change that. Should it be 4 spaces per indentation?

Reply:
Have you tried using the Prettier extension with VS code? That's the only way I can figure out formatting :)